doc/tutorial/02_custom_metrics.md

```diff
@@ -0,0 +1,34 @@
+# Custom metrics for pgexporter
+
+This tutorial will show how to set custom metrics for pgexporter.
+
+## Preface
+
+This tutorial assumes that you have an installation of PostgreSQL 10+ and pgexporter.
+
+For RPM based distributions such as Fedora and RHEL you can add the
+[PostgreSQL YUM repository](https://yum.postgresql.org/) and do the install via
+
+```
+dnf install -y postgresql10 postgresql10-server pgexporter
+```
+
+
+## Change pgexporter configuration
+
+Change `pgexporter.conf` to add 
+
+```
+metrics_path=[PATH_TO_YAML_FILE_DIR]
+```
+
+or run the command with `-Y` options.(`pgexporter` user)
```

Comment:
Missing line feed before the user description

Reply:
I don't understand this clearly.